_posts/2020-10-13-community-call-dei.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: Summary of October 2020 Community Call - US-RSE's Role in Promoting Diversity, Equity and Inclusion
+tags: [community call]
+posted_by: Nicole Brewer, Daniel S. Katz, Sandra Gesing, Ian Cosden, Chris Hill
+---
+
+On September 10, 2020, the US-RSE Organization held a community call on "US-RSE's role in promoting Diversity, Equity and Inclusion". The goal of the call was to have an open discussion about what steps US-RSE might take in promoting diversity in our profession, both within the US-RSE organization and externally through outreach to the broader community, and if the participants in the call supported the idea of taking such steps.
```

Comment:
```suggestion
On September 10, 2020, the US-RSE Organization held a community call on "US-RSE's role in promoting Diversity, Equity and Inclusion." The goal of the call was to have an open discussion about what steps US-RSE might take in promoting diversity, both within the US-RSE organization and externally through outreach to the broader community. 
```
Suggested change. I don't think there was any question about if people supported this, so I removed the last sentence.

Reply:
I agree that the people on the call supported it, but I'm not sure that the whole community does.  Maybe this would be better as
> The goal of the call was to have an open discussion about what steps US-RSE might take in promoting diversity within the US-RSE organization and externally in our profession through outreach to the broader community, and to determine if the participants in the call supported the idea of taking such steps.